lib/review_bot/reviewer.rb

```diff
@@ -16,5 +16,9 @@ def work_hour?
     def slack_emoji
       ":#{slack}:"
     end
+
+    def not_user_slack_emoji
+      ":#{not_user_slack}:"
```

Comment:
Do we need this to be part of the config? Or is something like this ok?

```suggestion
      ":not_#{slack}:"
```

Reply:
I've went back and forth on that. Part of me wanted the user to be able to put in whatever name they wanted in the config rather than forcing the naming for the emoji from inside the gem. But I could be convinced either way.

Reply:
🤔 maybe we allow for a custom `not` emoji, but the implementation uses `not_#{slack}` as the default, so it's optional? and most of the time it will "just work"?